modules/ui/tag_reference.js

```diff
@@ -10,53 +7,61 @@ import { t } from '../util/locale';
 import { utilDetect } from '../util/detect';
 import { services } from '../services';
 import { svgIcon } from '../svg';
+import { utilQsString } from '../util';
 
 
 export function uiTagReference(tag) {
-    var taginfo = services.taginfo;
+    var wikibase = services.osmWikibase;
     var tagReference = {};
 
     var _button = d3_select(null);
     var _body = d3_select(null);
     var _loaded;
     var _showing;
 
-
+    /**
+     * @returns {{itemTitle: String, description: String, image: String|null}|null}
+     **/
     function findLocal(data) {
-        var locale = utilDetect().locale.toLowerCase();
-        var localized;
-
-        if (locale !== 'pt-br') {  // see #3776, prefer 'pt' over 'pt-br'
```

Comment:
For the benefit of others reading this PR, I’m pretty sure the MediaWiki API handles #3776 and other language fallbacks implicitly.